tests/test_burst.py

```diff
@@ -65,27 +39,31 @@ def test_spoof_safe(tmp_path, mocker, pattern):
     assert len(list(tmp_path.glob(pattern))) == 1
 
 
-@pytest.mark.parametrize('orbit', ('ascending', 'descending'))
-def test_get_region_of_interest(orbit):
-    options = {'descending': REF_DESC, 'ascending': REF_ASC}
+@pytest.mark.parametrize('orbit', ('ascending', 'ascending'))
```

Comment:
Shouldn't one of these still be `descending`?

```suggestion
@pytest.mark.parametrize('orbit', ('ascending', 'descending'))
```